### ISSUE TITLE
Is p7zip affected by remote code execution security vulnerabilities of "normal" 7zip?

### ISSUE THREAD

**Issue by therealmarv** [p7zip-project/p7zip#224]
Seems 7zip 22.01 from Igor Pavlov was affected by these two new found security bugs:

* [CVE-2023-40481](https://vuldb.com/?id.237882)
* [CVE-2023-31102](https://vuldb.com/?id.237881)

It seems those issues **allow remote code execution** by opening files !!!

7zip has released new versions (23.01) which apparently fix those issues.

If p7zip does not react to new found 7zip security issues in time I consider it as Abandonware.
Unfortunately p7zip is the default on many Linux distros out there.

Thinking out loud: Maybe it's time to retire p7zip now that Igor Pavlov releases official Mac&Linux source and binaries nowadays.

**Comment by tansy** [comment]
> Seems 7zip 22.01 from Igor Pavlov was affected by these two new found security bugs:
> 
>     * [CVE-2023-40481](https://vuldb.com/?id.237882)
> 
>     * [CVE-2023-31102](https://vuldb.com/?id.237881)
> 
> 
> It seems those issues **allow remote code execution** by opening files !!!
> 
> 7zip has released new versions (23.01) which apparently fixes those issues.
> 
> Unfortunately p7zip is the default on many Linux distros out there.
> 
> Any statements about those two security issues?

Yes. You can go to sf.net/p/sevenzip, download and compare versions 22.01 and 23.01, get what it does to solve the issue, incorporate it to 17.05, request a pull and get an award for being security hero.

**Comment by tansy** [comment]
What's interesting, Pavlov [says](https://sourceforge.net/p/sevenzip/support-requests/498/#20ea) that
> p7zip 16.02 is not affected by CVE-2023-31102
